src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectsRestClient.java

```diff
@@ -90,4 +91,24 @@ public ProjectApi name(String name) throws RestApiException {
         }
         return projectsParser.parseProjectInfos(result);
     }
+
+    @Override
+    public ProjectApi create(String name) throws RestApiException {
+        ProjectInput projectInput = new ProjectInput();
+        projectInput.name = name;
+        return this.create(projectInput);
+    }
+
+    @Override
+    public ProjectApi create(ProjectInput in) throws RestApiException {
+        if(in.name == null) {
```

Comment:
Please add a space after `if`. Also, I think a `NullPointerException` would fit better here. AFAIK `AssertionError` is better not used in such places when we rely on user-input. See also: http://stackoverflow.com/a/24863229